src/test/components/containers/editProfileContainer/__snapshots__/EditProfileContainer.test.js.snap

```diff
@@ -0,0 +1,342 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
#### Critical
- Do the needful and `.gitignore` ___snapshots___

Reply:
We are recommended to commit the snapshot and add to our version control system.
https://jestjs.io/docs/en/snapshot-testing.html#are-snapshots-written-automatically-on-continuous-integration-ci-systems